integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java

```diff
@@ -1162,69 +1254,159 @@ private static void installWebHookAndAlertManager() throws Exception {
 
     crdCmd = "kubectl apply -f " + monitoringExporterEndToEndDir + "/webhook/server.yaml";
     TestUtils.exec(crdCmd);
-    String webhookPod = getPodName("webhook", "webhook");
+    String webhookPod = getPodName("app=webhook", "webhook");
     TestUtils.checkPodReady(webhookPod, "webhook");
 
   }
 
   /**
-   * Uninstall Prometheus and Grafana using helm chart.
+   * Install WebHook for performing scaling via prometheus
+   *
+   * @throws Exception if could not run the command successfully to install webhook and alert manager
+   */
+  private static void createWebHookForScale() throws Exception {
+
+    String  webhookResourceDir = resourceExporterDir + "/../webhook";
+    String  webhookDir = monitoringExporterDir + "/webhook";
+    if (!new File(webhookDir).exists()) {
+      Files.createDirectories(Paths.get(webhookDir));
+      logger.info("webhookDir" + webhookDir);
+      logger.info("webhookResourceDir" + webhookResourceDir);
+      String crdCmd = "cp -rf " + webhookResourceDir + "/* " + webhookDir;
+      logger.info("executing command " + crdCmd);
+      TestUtils.exec(crdCmd);
```

Comment:
As you are calling TestUtils.exec multiple times in this method, how about putting all these in one shell script and call that script? except for the kubectl checks..

Reply:
replaced with shell script to create/install web hook

Reply:
Hi Marina,
Can u add simile script for the mvn/docker/kubectl command used in 
buildMonitoringExporter()  and deletePrometheusGrafana()

Reply:
I am working on https://jira.oraclecorp.com/jira/browse/OWLS-75941, will switch to shell scripts there